jina/peapods/compoundpod/__init__.py

```diff
@@ -308,15 +309,15 @@ def wait_start_success(self) -> None:
             self.close()
             raise
 
-    def _enter_replica(self, replica: 'BaseReplica') -> None:
+    def _enter_replica(self, replica: 'Pod') -> None:
         self.replica_list.append(replica)
         self.enter_context(replica)
 
-    def _enter_pea(self, pea: 'BasePea') -> None:
+    def _enter_pea(self, pea: 'Pea') -> None:
         self.peas.append(pea)
         self.enter_context(pea)
 
-    def __enter__(self) -> 'BasePod':
+    def __enter__(self) -> 'Pod':
```

Comment:
it is a `CompoundPod` the return type right?

Reply:
yes thanks